src/Card/Service.php

```diff
@@ -24,11 +24,4 @@ public function find(string $uid): Card;
      * @return Card
      */
     public function create(Token $token = null): Card;
-
-    /**
-     * Deletes a card.
-     *
-     * @param Card $card
-     */
-    public function delete(Card $card): void;
```

Comment:
Moving stuff closer to a Customer as you'll always need to perform this kind of action with a Customer makes sense.

Reply:
FYI You actually won't need a customer with the new Payment Intent Api when you're removing a Payment method which is why it was seperated. But it's just way easier to use that way.

Reply:
I don't think we should have changed this...

Reply:
Why? Backward compatibility issues?

Reply:
So there is two delete methods one for the Source API cards and one for the new Payment Method API. I regrouped both in one but wasn't a good idea in the end.